core/domain/event_services_test.py

```diff
@@ -290,6 +532,13 @@ def setUp(self):
             self.save_new_valid_exploration('exp_id', self.admin_id))
 
     def test_average_ratings_of_users_exps_are_calculated_correctly(self):
+
+        # If user id is not present in UserStatsModel.
```

Comment:
I don't quite understand this comment. How does what the comment says connect to the code that follows? Perhaps explain the link a bit more.

Reply:
@seanlip I have updated the comment and also toggled the code just to make it more clear!
PS: replaced `self.admin_id` with `'invalid_id'`

Reply:
Actually, I misinterpreted the code. I have removed this comment. It is not at all required.

Reply:
Instead of this, We can just write: `# If we are rating for the first time.` but that is understood.